sycl/source/detail/device_info.hpp

```diff
@@ -764,6 +764,8 @@ inline cl_ulong get_device_info_host<info::device::global_mem_cache_size>() {
   return PlatformUtil::getMemCacheSize();
 }
 
+// max_constant_buffer_size and max_constant_args are deprecated in SYCL 2020
+#ifdef __SYCL_INTERNAL_API
```

Comment:
Why not remove deprecated code as you did for other deprecated API?
I think we must be consistent with deprecating/removing unsupported API.

Reply:
The plan for deprecating features was proposed next way : wrap deprecated API to the __SYCL_INTERNAL_API macro, then check that API is not available without this macro and check that API still working with this macro. Apparently for sycl::byte and handler::codeplay_host_task it was decided that these API could be fully removed without wrapping.

Reply:
handler::codeplay_host_task is template method and can be removed cause its realization lies in user's executable. It was decided to delete sycl::byte in discussion about this alias. The point of macro is to keep ABI the same, but remove API. I clarified it in the description for new macro